src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -301,7 +302,7 @@ struct PrescribedAtmosphere{FT, M, G, T, U, P, C, F, I, R, TP, TI}
     downwelling_radiation :: R
     thermodynamics_parameters :: TP
     times :: TI
-    reference_height :: FT
+    surface_layer_height :: FT
     boundary_layer_height :: FT
```

Comment:
I believe this should be in the `flux_formulation` because it is used only for calculating the gustiness in the `Simi;arityTheoryFluxes`. Maybe for another PR, but to keep in mind

Reply:
in principle this is a diagnostic variable that must be computed from the atmospheric state